### ISSUE TITLE
Input reference image have to come from a rendering of the gt_scene?

### ISSUE THREAD

**Issue by VincentHancoder** [jkxing/DROT#5]
Hi author, nice work! When I use your code, I notice that your reference image `gt_img` and `gt_img_torch` come from the rendering of `gt_scene`. I wonder if it is possible to only use `0_gt.png` as a reference image to guide the lay-out process. Because in practice, I tend to only have one correct image (without camera pose) instead of gt_scene. Feel free to correct me if I've misinterpreted anything.🤝

**Comment by jkxing** [comment]
Of course! You can use your image. Make sure that the color range and image resolution fit the requirements.
